felipe_test.py

```diff
@@ -0,0 +1,47 @@
+import boto3
```

Comment:
This file is not necessary, if all the tests are already covered in `tests/test_logs`